HSDS/Reader/Stats.h

```diff
@@ -46,12 +46,12 @@ class Stats {
     return convert_to_vector();
   }
 
-  void collect_datapoint()
+  void collect_datapoint(const OpenDDS::DCPS::SystemTimePoint& now)
```

Comment:
Thanks for updating this! I was about to send a note to Daniel that the timestamps can be different between different topics given the current code. But this change makes them exactly the same for sure.